### ISSUE TITLE
Merge OpenFAST FSI branch into develop

### ISSUE THREAD

**Issue by psakievich** [psakievich/hfm-fsi#53]

**Comment by gantech** [comment]
Got compile errors down to Actuator line issues. Trying to merge soon.. hopefully within 1/2 days

**Comment by kevmoor** [comment]
Finished, with Andy and Derek's help, works with latest dev branch and standalone openFAST driver (but not tested the C++ api).  Only outstanding thing is one regression test needs to be updated.  Once those are done, should be able to merge into dev.  But, won't allow you to use AMR wind coupled to openfast with actuator line, but will work with the NALU coupling.  Question of if the latest dev branch makes a difference in terms of beamdyn fixes.  Should be able to see this in standalone openfast cases. If not happening, no difference will be seen for FSI.  Once C++ api is ready/tested, should be ready for team use, is currently in Andy Platt's public facing fork: https://github.com/andrew-platt/openfast/tree/f/ExtLoads_rebase.  Should be merged today into OpenFAST Dev, will ping on slack when its merged. IEA 15 MW models now work with the dev branch as well.

**Comment by pscrozi** [comment]
Ganesh made changes and got it working, but hasn't merged yet. C++ api is working.

**Comment by psakievich** [comment]
Need to make sure the timestep output is correct. PR is up https://github.com/OpenFAST/openfast/pull/1932
